### ISSUE TITLE
Hi! I'm trying to use your awesome library GroupDocs.Watermark for removing watermark from pdf file. But in evaluation mode this operation is not available. How can see removing of one watermark in my file without buying license.

### ISSUE THREAD

**Issue by frigoshor** [groupdocs-free-consulting/projects#28]
Please check if it satisfies the following requirements: 

* It is a small project satisfying a particular scenario
* It is related to GroupDocs APIs
* It takes 8 or less hours for us to complete
* It is not a bug
* It is not a feature request

If your requirement is actually a bug in our APIs or a feature request for our APIs then please use our forum to report it and get quick help: https://forum.groupdocs.com/

If all the above requirements are satisfied then please move on:

< describe your free consulting project requirements briefly>

< which platform you're working on -- operating system, development environment etc.> 

<attach input documents>

<attach any sample required output documents if you have> 

<attach any additional supporting material to help us understand the requirement>

**Comment by atirtahirgroupdocs** [comment]
@frigoshor 

Yes, there are evaluation limitations. But you can request a [temporary license](https://purchase.groupdocs.com/temporary-license). A temporary license is a time-restricted full license. Please follow these [steps](https://kb.groupdocs.com/total/net/how-to-get-a-temporary-license-for-groupdocs/) to avail/request the license.
